openapi/single-sign-on/single-sign-on.yaml

```diff
@@ -0,0 +1,384 @@
+        - schema:
+            type: string
+          in: cookie
+          name: vig-namespaced-session
+          description: 'Cookie for a Vendasta Identity Gateway session, stored at namespaced paths.'
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: id_token_hint
+          description: '[spec miss: Identity token of user requesting logout, used to authentically indicate user and service requesting logout. Required if using `post_logout_redirect_uri`.] '
+      tags:
+        - Single-Sign-On
+      security: []
+    parameters:
+      - schema:
+          type: string
+        name: partnerID
+        in: path
+        required: true
+        description: Partner ID providing identity namespacing
+  '/{partnerID}/oauth2/token':
+    parameters:
+      - schema:
+          type: string
+        name: partnerID
+        in: path
+        required: true
+        description: Partner ID providing identity namespacing
+    post:
+      summary: OAuth2 Token
+      operationId: openid-token
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: Access token with the requested and granted scopes.
+                type: object
+                x-examples:
+                  example-1:
+                    access_token: SlAV32hkKG
+                    token_type: Bearer
+                    refresh_token: 8xLOxBtZp8
+                    expires_in: 3600
+                    id_token: eyJhbGciOiJSUzI1NiIsImtpZCI6IjFlOWdkazcifQ
+                properties:
+                  access_token:
+                    type: string
+                    minLength: 1
+                    description: 'Token Type being returned, '
+                  token_type:
+                    type: string
+                    minLength: 1
+                    enum:
+                      - bearer
+                    description: 'OAuth 2.0 Token Type value, will be `bearer`, indicating access_token is to be used as bearer token.'
+                  refresh_token:
+                    type: string
+                    minLength: 1
+                    description: A refresh token which can be used as an authorization grant for delegated access when user offline or not using the application. Only returned when requested with `offline_access` scope.
+                  expires_in:
+                    type: number
+                    description: |
+                      Expiry of Access Token in seconds.
+                  id_token:
+                    type: string
+                    minLength: 1
+                    description: 'Identity Token containing base user identity data (user id) and extra data based on requested and granted scopes (email, name). To get this data from an access token, use the UserInfo endpoint.'
```

Comment:
This would benefit from a pointer to unpack identity token vs user info and what data is there and how to get access to it by scopes